src/Autofac/Builder/RegistrationBuilder{TLimit,TActivatorData,TRegistrationStyle}.cs

```diff
@@ -262,6 +278,20 @@ public RegistrationBuilder(Service defaultService, TActivatorData activatorData,
             return this;
         }
 
+        /// <summary>
+        /// Configure the services that the component will provide.
+        /// </summary>
+        /// <param name="service">Services to expose.</param>
+        /// <returns>A registration builder allowing further configuration of the component.</returns>
+        public IRegistrationBuilder<TLimit, TActivatorData, TRegistrationStyle> As(Service service)
```

Comment:
No problem here. I think the `AddService` method has only been used by internal registration sources but has always been part of the public API.